addons/account/models/account_move.py

```diff
@@ -3171,6 +3171,10 @@ def button_draft(self):
     def button_cancel(self):
         self.write({'auto_post': False, 'state': 'cancel'})
 
+    def is_mail_sendable(self):
```

Comment:
This should not be a public method and the name should be more clear.  _is_sent_by_mail_type or something

Reply:
_can_be_emailed is now in v16 standard